pull-report.js

```diff
@@ -167,8 +204,10 @@ if (require.main === module) {
     .option("--gh-user <username>", "GitHub user name", null)
     .option("--gh-pass <password>", "GitHub pass", null)
     .option("--gh-token <token>", "GitHub token", null)
-    .option("--pr-url", "Add pull request URL to output", false)
+    .option("--pr-url", "Add pull request or issue URL to output", false)
     .option("--repo-type <type>", "Repo type (default: all|member|private)", "all")
+    .option("--noprs", "Do not display pull requests", false)
```

Comment:
This doesn't seem to match the README.md documentation. 

I think you can just add `--pull-requests` with default `true` and updated message. And I _think_ the `--no-pull-requests` is then automagically provided, but we should check this.